### DISCUSSION TITLE
Unable to be seen or accessed, the workflow is on the portal engine page for the data object.

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#15472] [Q&A / Support]
Hi team,

I have added a new Asset workflow to my enterprise Pimcore project, and I can see the workflow popup and access it from the portal engine page. I similarly created the data objects, but I am unable to view the workflow popup or access them on the portal engine page. They are functional on the admin page, though.

Workflow Config for Asset :-
![image](https://github.com/pimcore/pimcore/assets/106297450/79a2674f-86c5-4617-bde9-d6ed16621d72)
![Screenshot 2023-07-04 115512](https://github.com/pimcore/pimcore/assets/106297450/9ff41b39-0382-4312-9d59-e819cc75c83c)

Asset Workflow popup in PXP page:-
![Screenshot 2023-07-04 113626](https://github.com/pimcore/pimcore/assets/106297450/0c8aaabc-cba4-470f-886b-ab0f5dedf6fa)

Workflow Config for Object :-
![Screenshot 2023-07-04 115630](https://github.com/pimcore/pimcore/assets/106297450/a462501f-82b7-4235-aebc-332a95e926fe)
![Screenshot 2023-07-04 115649](https://github.com/pimcore/pimcore/assets/106297450/8e47ee91-226c-4e3f-ab6b-40bf1795102f)

The DataObject Workflow popup failed to locate it.

For a data object, how do I achieve this?

**Comment by arulselvan008** [comment]
@kingjia90 @fashxp 

Kindly help us achieve this.

**Comment by fashxp** [comment]
If you have time critical questions, please contact your enterprise support contact.
